core/src/test/java/io/confluent/rest/SaslTest.java

```diff
@@ -1,4 +1,4 @@
-/**
```

Comment:
nit, existing files have two stars, we should probably put this into checkstyle

Reply:
two stars is incorrect. this is not a java doc.